src/components/badges/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { FC, ReactNode } from "react"
+
+import classNames from "classnames"
+
+import styles from "./badges.module.css"
+
+import Tooltip, { TooltipPosition } from "../tooltip"
+import { Language } from "../../types"
+
+export type BadgeSize = "small" | "large"
+export interface BadgeProps {
+  size: BadgeSize
+  text: string
+  icon: ReactNode
+  tooltipContent?: ReactNode
+}
+export interface ExternalBadgeProps {
```

Comment:
why are the external? do you distinguish between internal/external ones from the pov of the package? 
maybe lend from our Foo/FooBase pattern that we often apply? Badge for the external BadgeBase for the shared one?

Reply:
I also feel that this type is misplaced. it has nothing to do with the component it is encapsulated with

Reply:
- Shared badge types no live in `badges/types.ts`
- Index has been renamed to base